style-transfer/libs/numpy.js

```diff
@@ -0,0 +1,299 @@
+/* jshint esversion: 6 */
```

Comment:
we have numpy.js in https://github.com/webmachinelearning/webnn-samples/blob/master/nsnet2/libs/numpy.js, could you please reuse that one? You may move it to a top libs and share it with both nsnet2 and style-transfer examples.